recipes/pvxs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.3.3" %}
+
+package:
+  name: pvxs
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/epics-base/pvxs/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 944d16687fbdd31f43c617824048891756ff9edd93d209df260b17d2adc0b446
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+  run_exports:
+    - libevent
+    - {{ pin_subpackage('pvxs', max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - epics-base
+    - libevent
+
+test:
+  commands:
+    - pvxget -V
```

Comment:
Comparing to epics-base-feedstock, maybe test each of the generated executables? https://github.com/conda-forge/epics-base-feedstock/blob/main/recipe/meta.yaml#L39